helmfile/values/fluentd-wc.yaml.gotmpl

```
@@ -3,6 +3,11 @@ nodeSelector: {{- toYaml .Values.fluentd.nodeSelector | nindent 2  }}
 affinity:     {{- toYaml .Values.fluentd.affinity | nindent 2  }}
 tolerations:  {{- toYaml .Values.fluentd.tolerations | nindent 2  }}
 
+
+image:
+  repository: elastisys/fluentd
+  tag: 3.4.5
```

Maybe it would be better to not increase the version of the upstream image but rather add a postfix (as we do with kubespray etc.)
```suggestion
  tag: v3.4.0-ck8s1
```